### PR TITLE
ISPN-7228 Creating wrong schema doesn't show error message

### PR DESCRIPTION
**Pull request by vblagoje**
Master only. See server setup needed to test this error in ISPN-7402. Once ISPN-7402 and ISPN-7405 fixes are integrated into the server this issue should be resolved automatically.

### CODE REVIEW

**Comment by vblagoje** [comment]
@ryanemerson apologies for not being clear. You need to update your server with latest head and then apply patches referenced in https://issues.jboss.org/browse/ISPN-7402 Only then you can try this one out :-(

**Comment by ryanemerson** [comment]
@vblagoje Ok, I will review this again once the fixes in adrian's PR have been integrated.

**Comment by vblagoje** [comment]
Ok @ryanemerson lets wait for https://issues.jboss.org/browse/ISPN-7402 and https://issues.jboss.org/browse/ISPN-7405 to be integrated!

**Comment by vblagoje** [comment]
@ryanemerson you can integrate it now as the error is being raised and displayed in UI. After a solution for https://issues.jboss.org/browse/ISPN-7405 is integrated we'll get the details of the error as well! You'll need the latest server head or CR1.

File: src/services/schemas/SchemasService.ts [hunk 1]
```diff
@@ -62,7 +62,11 @@ export class SchemaService {
           });
         },
         error => deferred.reject(error))
-      .then(() => deferred.resolve(), error => deferred.reject(error));
+      .then(() => {
+        this.getProtoSchemaErrors(container, schema.fileName).then((error) => {
```
Comment (ryanemerson) [inline src/services/schemas/SchemasService.ts]: You no longer resolve the deferred promise, so this doesn't work in the normal case.

**Comment by vblagoje** [comment]
Updated with a new commit addressing an issue you noticed @ryanemerson

**Comment by ryanemerson** [comment]
@vblagoje Using CR1 this works when creating a correct schema, but no error messages are displayed in the console. I do get lots of errors on the server though :/

**Comment by vblagoje** [comment]
@ryanemerson Yeah you need to remove <globalstate> from your configuration. Or wait for https://github.com/infinispan/infinispan/pull/4868 to get integrated :-)

**Comment by vblagoje** [comment]
@ryanemerson https://github.com/infinispan/infinispan/pull/4868 got integrated so you can test this one out of the box with the latest infinispan server

**Comment by ryanemerson** [comment]
Integrated fa7ca6170a385424350a9cf0f9d9369b43bc6413, thanks!

**Pull request closed without merging**